packages/core/helper-plugin/src/features/Notifications.tsx

```diff
@@ -221,65 +229,16 @@ const Notification = ({
     >
       {formatMessage(
         {
-          id: message?.id || message,
-          defaultMessage: message?.defaultMessage || message?.id || message,
+          id: typeof message === 'object' ? message.id : message,
```

Comment:
we can have a `messageObject = typeof message === 'object' ? message : null;` to avoid checking for each prop

Reply:
we could, but i think this is probably a bit more specific and keeps current behaviour so smaller risk of breaking anything 🤞🏼